docs/ug/usingAuthorTags.md

```diff
@@ -32,6 +32,25 @@ The `@@author` tags should be enclosed within a comment, using the comment synta
 
 ![author tags](../images/add-author-tags.png)
 
+Currently, the following comment formats are supported:
+<ul>
+    <li>// @@author authorName</li>
+    <li>/* @@author authorName */</li>
+    <li>/* @@author authorName</li>
```

Comment:
I just try to make sure that if they use multiple lines comment, the author tag must be on the first line. The author tag can be in another comment block after the code snippet and it can be any of '// @@author', '/* @@author' or '/* @@author'. If the implementation for author tag in multi-line comment is ambiguous, may be we can consider allowing the '@@author' tag in inline comment only?